FABulous/FABulous.py

```diff
@@ -869,10 +870,50 @@ def do_gen_geometry(self, *vargs):
         if 4 <= padding <= 32:
             self.fabricGen.genGeometry(padding)
             logger.info("Geometry generation complete")
-            logger.info(f"{geomFile} can now be imported into the FABulous Editor")
+            logger.info(f"{geomFile} can now be imported into FABulator")
         else:
             logger.error("padding has to be between 4 and 32 inclusively!")
 
+    def do_start_FABulator(self, *ignored):
+        """Starts FABulator if an installation can be found.
+        If no installation can be found, a warning is produced.
+
+        Usage:
+            start_FABulator
+
+        Parameters
+        ----------
+        *ignored : tuple
+            Ignores additional arguments.
+
+        """
+        logger.info("Checking for FABulator installation")
+        fabulatorRoot = os.getenv("FABULATOR_ROOT")
+        if fabulatorRoot is None:
+            logger.warning("FABULATOR_ROOT environment variable not set.")
+            logger.warning(
+                "Install FABulator and set the FABULATOR_ROOT environment variable to use this feature."
+            )
+        elif not os.path.exists(fabulatorRoot):
+            logger.warning(
+                f"FABULATOR_ROOT environment variable set to {fabulatorRoot} but the directory does not exist."
+            )
```

Comment:
You can up the logging the error and do a return to terminate the command early.